README.md

```diff
@@ -1 +1,178 @@
+- HTML file 
+- CSS file 
+- JS files
+- Html,CSS and JS linters file
+
+
+The repository also contains files for setting up linters and react-app.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+  <ul>
+    <li><a href="https://microverse.notion.site/HTML-CSS-Get-a-head-start-275eb85fd34b4416aa06ec635d69cdaf">HTML</a></li>
+    <li><a href="https://microverse.notion.site/HTML-CSS-Get-a-head-start-275eb85fd34b4416aa06ec635d69cdaf">CSS</a></li>
+       <li><a href="https://microverse.notion.site/HTML-CSS-Get-a-head-start-275eb85fd34b4416aa06ec635d69cdaf">React-JS</a></li>
+  </ul>
+
+
+
+### Key Features <a name="key-features"></a>
+## Math Magician - [HTML/CSS & REACT Math Magician Project](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes_v1_1.md)
+### Features Added:
+- Create a calculator ui with React-js.
+- Create all functionality a calculator have with react state and props.
+- Create a (SPA) single page app but behave like multiple page app.
+- Display a quote every time you go on quote page from the api.
+- CSS styles Added to the application to make it match [this](https://github.com/microverseinc/curriculum-javascript/blob/main/books/images/math_magicians_core_elements.png) wireframe.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+> cd my-folder
+> git clone git@github.com:myaccount/my-project.git
+
+### Prerequisites
+
+In order to run this project you need:
+
+- gitHub account;
+- git installed on your OS.
+
+### Install
+> https://github.com/microverseinc/linters-config/tree/master/html-css-js
+- Installations required to run this project:
+
+### Webhint installation. 
+- Run the following command:
+> npm install --save-dev hint@7.x
+
+### Stylelint installation.
+- Run the following command:
+> npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+### ESLint
+- Run
+
+> npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+
+### Usage
+
+You can use this project by cloning it to your folder and changing index.html and styles.css files.
+
+### Run tests
+
+To run tests, run the following commands:
+
+To track linter errors locally follow these steps:  
+
+Download all the dependencies run:
+```
+npm install
+```
+Track HTML linter errors run:
+```
+npx hint .
+```
+Track CSS linter errors run:
+```
+npx stylelint "**/*.{css,scss}"
+```
+Track JavaScript linter errors run:
+```
+npx eslint ."
+```
+To Open The Project On The Browser
+npm start
+```
+
```

Comment:
- [ ] Kindly remove these three backslash dots on Line 133 so that the rest of the README.md file can look professional.

### Current Display

![image](https://github.com/jubaer919/math-magicians/assets/77106962/dd65f779-5317-4f5a-a534-35e3eb2a12e2)